modules/jdbc/src/main/java/org/testcontainers/containers/JdbcDatabaseContainer.java

```diff
@@ -45,6 +45,11 @@ public JdbcDatabaseContainer(String dockerImageName) {
      */
     public abstract String getJdbcUrl();
 
+    /**
+     * @return the database name
+     */
+    public abstract String getDatabaseName();
```

Comment:
if we add abstract method we break the backward compatibility (because every child of this class must implement this method). How about implementing it for now with some "not implemented" exception?

Reply:
https://github.com/testcontainers/testcontainers-java/pull/520/commits/df9b02dc9c13ea7f2d9d07b5d9063ea7dc905665